docs/appendices/release-notes/5.7.1.rst

```diff
@@ -46,4 +46,8 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed an issue that caused exceeding the
+  :ref:`cluster.max_shards_per_node <cluster.max_shards_per_node>` when running
+  ``INSERT INTO... VALUES...`` statement with multiple values on partitioned
+  tables.
+
```

Comment:
Maybe should say that can exceed significantly, basically bound only to VALUES count: the more values are provided, the bigger the overshooting becomes.

Reply:
Given that exceeding the limit is [by design](https://github.com/crate/crate/issues/15803#issuecomment-2058729661) for tables with `number_of_replicas` given as a range (and range 0-1 is our default value) we probably should emphasize that here overshooting can be significant and is bound only to VALUES count: the more values are provided, the bigger the overshooting becomes.